presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -494,6 +494,17 @@ public void renameColumn(ConnectorSession session, ConnectorTableHandle tableHan
         HiveTableHandle hiveTableHandle = checkType(tableHandle, HiveTableHandle.class, "tableHandle");
         HiveColumnHandle sourceHandle = checkType(source, HiveColumnHandle.class, "columnHandle");
 
+        Optional<Table> table = metastore.getTable(hiveTableHandle.getSchemaName(), hiveTableHandle.getTableName());
```

Comment:
I will move this change back to `BridgingHiveMetastore` since I am making some other changes.